app/gps/intersect/toExposureHistory.ts

```diff
@@ -0,0 +1,60 @@
+import {
```

Comment:
I might call this file, `exposureHistory` rather than `toExposureHistory` its better to be grouping code by domain concept rather than function name